client/src/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-
+import { Route } from 'react-router-dom';
```

Comment:
Well done, I'm asking everyone if the idea of destructuring the import and when not to makes sense yet.

Can you explain it?